python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -1038,10 +1038,7 @@ def _create_sensor_run(
     execution_plan_snapshot = external_execution_plan.execution_plan_snapshot
 
     job_tags = validate_tags(external_job.tags or {}, allow_reserved_tags=False)
-    tags = merge_dicts(
```

Comment:
actually we need to keep this- otherwise old sensor code on new daemons won't get the tag